src/main/java/com/pri/cabzza/web/rest/PortfolioStoreResource.java

```diff
@@ -0,0 +1,134 @@
+package com.pri.cabzza.web.rest;
+
+import com.codahale.metrics.annotation.Timed;
+import com.pri.cabzza.domain.PortfolioStore;
+import com.pri.cabzza.repository.PortfolioStoreRepository;
+import com.pri.cabzza.repository.search.PortfolioStoreSearchRepository;
+import com.pri.cabzza.web.rest.util.HeaderUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.inject.Inject;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import static org.elasticsearch.index.query.QueryBuilders.*;
+
+/**
+ * REST controller for managing PortfolioStore.
+ */
+@RestController
+@RequestMapping("/api")
+public class PortfolioStoreResource {
```

Comment:
this name should follow controller naming convention. Maybe PortfolioStoreController?